runestone/mchoice/test/test_assess.py

```diff
@@ -1,16 +1,37 @@
-"""
-Test Multiple Choice question directive
-"""
+# ****************************************************
+# |docname| - Test Multiple Choice question directive
+# ****************************************************
 
 __author__ = "yasinovskyy"
 
 from unittest import TestCase
-from selenium.webdriver.support import expected_conditions as EC
 from selenium.webdriver.common.by import By
 from runestone.unittest_base import module_fixture_maker, RunestoneTestCase
 
 mf, setUpModule, tearDownModule = module_fixture_maker(__file__, True)
 
+
+class element_has_css_class:
```

Comment:
Ooooh nice!

Reply:
Actually this seems like it would be good to have in unittest_base so that we can use it in other tests

Reply:
Good point. I'll move it there.